src/components/Confirm/index.js

```diff
@@ -25,30 +25,32 @@ const RetakeAction = localised(({retakeAction, translate}) =>
   </Button>
 )
 
-const ConfirmAction = localised(({confirmAction, translate, error}) =>
+const ConfirmAction = localised(({confirmAction, btnDisabled, translate, error}) =>
```

Comment:
Minor: I would consider renaming `btnDisabled` to `isUploading` since I don't see a problem with these component being aware that an upload is happening

Reply:
and renamed `btnDisabled` to `isUploading`